src/core/atom_component/dragTree/component/treeNode/SceneTreeNode.re

```diff
@@ -48,12 +73,43 @@ module Method = {
     DragEnd;
   };
 
-  let handleDrop =
+  let handleDragOver =
       (id, (isWidgetFunc, checkNodeRelationFunc, isAssetWDBFileFunc), event) => {
     let e = ReactEventType.convertReactMouseEventToJsEvent(event);
+    EventHelper.preventDefault(e);
+
+    let (isTrigger, _) =
+      DragEventBaseUtils.checkDragEnter(
+        id,
+        isWidgetFunc,
+        checkNodeRelationFunc,
+      );
+
+    let isSceneGameObject =
+      SceneEngineService.getSceneGameObject
```

Comment:
TODO refactor to "id === SceneEngineService.getSceneGameObject |> StateLogicService.getEngineStateToGetData"